pkg/kubelet/metrics/metrics.go

```diff
@@ -194,10 +201,13 @@ var (
 	// The buckets max value 40 is based on the 45sec max gRPC timeout value defined
 	// for the DRA gRPC calls in the pkg/kubelet/cm/dra/plugin/registration.go
 	DRADurationBuckets = metrics.ExponentialBucketsRange(.1, 40, 15)
+
+	// podResizeDurationBuckets is the bucket boundaries for pod_resize_duration_milliseconds metrics.
+	podResizeDurationBuckets = []float64{1, 5, 10, 20, 30, 60, 120, 180, 240, 300, 360, 480, 600, 900, 1200, 1800, 2700, 3600}
```

Comment:
open to opinions on what the buckets should be... I was looking at the results in my local kind cluster and the values seemed to hover somewhere between 20-200. Not sure if we expect it to be different on a real cluster?